<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Z100107" name="com.YouMagicPro.LKA_tests.OrderToAdressNoTextFieldFilledTest" tests="1" failures="0" timestamp="23 Jan 2013 08:36:26 GMT" time="23.147" errors="1">
  <testcase name="OrderToAdressNoTextFieldFilledTest_C25679" time="23.147" classname="com.YouMagicPro.LKA_tests.OrderToAdressNoTextFieldFilledTest">
    <error type="java.lang.AssertionError" message="expected [На вашем балансе недостаточно денег для совершения данной операции. Пожалуйста, пополните счет.] but found [Вы находитесь в состоянии финансовой блокировки. Пожалуйста, пополните счет.]">
      <![CDATA[java.lang.AssertionError: expected [На вашем балансе недостаточно денег для совершения данной операции. Пожалуйста, пополните счет.] but found [Вы находитесь в состоянии финансовой блокировки. Пожалуйста, пополните счет.]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:160)
	at com.YouMagicPro.LKA_tests.TestBase.assertEquals(TestBase.java:51)
	at com.YouMagicPro.LKA_tests.OrderToAdressNoTextFieldFilledTest.OrderToAdressNoTextFieldFilledTest_C25679(OrderToAdressNoTextFieldFilledTest.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- OrderToAdressNoTextFieldFilledTest_C25679 -->
</testsuite> <!-- com.YouMagicPro.LKA_tests.OrderToAdressNoTextFieldFilledTest -->
